lib/config/presets/internal/monorepo.ts

```diff
@@ -25,6 +25,7 @@ const repoGroups = {
   'azure azure-storage-net': 'https://github.com/Azure/azure-storage-net',
   'bugsnag-js': 'https://github.com/bugsnag/bugsnag-js',
   'chakra-ui': 'https://github.com/chakra-ui/chakra-ui',
+  'contentful-rich-text': 'https://github.com/contentful/rich-text',
```

Comment:
You can also use `contentful rich-text` if you'd like. I don't have a preference either way, and can merge this as-is if you'd prefer 👍

Reply:
@JamieMagee no whitespace and only lowercase. We have and issue for fixing older groups.

Reply:
TIL. Thanks for preventing me from making that mistake!

Reply:
fore reference #10210 🙃